src/main/java/com/checkmarx/flow/service/GitHubService.java

```diff
@@ -230,12 +240,10 @@ private Sources getRepoLanguagePercentages(ScanRequest request) {
                 log.warn(HTTP_BODY_IS_NULL);
             }
             return Arrays.asList(response.getBody());
-        }catch (NullPointerException e){
+        } catch (NullPointerException e) {
             log.warn(CONTENT_NOT_FOUND_IN_RESPONSE);
-        }catch (HttpClientErrorException.NotFound e){
-            log.error(ExceptionUtils.getStackTrace(e));
-        }catch (HttpClientErrorException e){
-            log.error(ExceptionUtils.getRootCauseMessage(e));
+        } catch (HttpClientErrorException e) {
+            log.error("Error getting repo content.", e);
```

Comment:
Consider if an error will occur here, the application will still continue to run... I had a case of NotFound exception in which I stopped the application by throwing an exception..

Reply:
Fixed.